code/datums/namepool.dm

```diff
@@ -71,3 +72,15 @@ GLOBAL_LIST_EMPTY_TYPED(namepool, /datum/namepool)
 /datum/namepool/robotic/get_random_name(gender = MALE)
 	. = pick(SSstrings.get_list_from_file(firstname_female_pool))
 	. += "-[rand(1,999)]" //pathfinder-738 or such
+
+/datum/operation_namepool
+	var/list/operation_titles = "names/operation_title"
+	var/list/operation_prefixes = "names/operation_prefix"
```

Comment:
Suggestion for after you finish this pr, you can remove this and make the operation name like tg's station names (see /proc/new_station_name())